### ISSUE TITLE
IPv6 / AAAA record

### ISSUE THREAD

**Issue by niklaskorz** [linkonoid/caddy-dyndns#12]
Currently there's no support for updating the AAAA record. I'd be happy to implement the required changes myself unless somebody is already working on it.

**Comment by linkonoid** [comment]
> Currently there's no support for updating the AAAA record. I'd be happy to implement the required changes myself unless somebody is already working on it.

Yes, IPv6 was not initially provided, since it is a matter of 100% support for this protocol from different providers (I'm not sure that all providers support it). You can make your changes to the repository to support this protocol for concrete provider (it is desirable to provide for the choice of protocol in the configuration file and leave the default IPv4).
